### ISSUE TITLE
LRN（Local Response Normalization） OP 尚未支持

### ISSUE THREAD

**Issue by tingggggg** [PaddlePaddle/X2Paddle#504]
如title
在轉換模型過程中，發現 tf.nn.lrn -> ONNX -> paddle
onnx2paddle 這段目前尚未支持

**Comment by Channingss** [comment]
LRN这个OP，ONNX和paddle1.8的时候功能实现不太一样，我们看一下paddle2.0是否一致。

**Comment by SunAhong1993** [comment]
> 如title
> 在轉換模型過程中，發現 tf.nn.lrn -> ONNX -> paddle
> onnx2paddle 這段目前尚未支持

已添加LRN op 转换相关代码，您可重装X2Paddle，进行转换，安装流程：
```
pip uninstall x2paddle
git clone https://github.com/SunAhong1993/X2Paddle.git
cd X2Paddle
python setup.py install
```
```

**Comment by tingggggg** [comment]
> > 如title
> > 在轉換模型過程中，發現 tf.nn.lrn -> ONNX -> paddle
> > onnx2paddle 這段目前尚未支持
> 
> 已添加LRN op 转换相关代码，您可重装X2Paddle，进行转换，安装流程：
> 
> ```
> pip uninstall x2paddle
> git clone https://github.com/SunAhong1993/X2Paddle.git
> cd X2Paddle
> python setup.py install
> ```

謝謝您
